gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/medication/MedicationMapperUtilsTest.java

```diff
@@ -136,4 +169,12 @@ private RCMRMT030101UKMedicationStatement unmarshallMedicationStatement(String f
     private RCMRMT030101UKEhrExtract unmarshallEhrExtract(String fileName) {
         return unmarshallFile(getFile("classpath:" + XML_RESOURCES_MEDICATION_STATEMENT + fileName), RCMRMT030101UKEhrExtract.class);
     }
+
+    @NotNull
+    private static INT createINTFromInteger(int value) {
+        INT i = new INT();
```

Comment:
Personal opinion - not a huge fan of single letter variable names! Could we call this `integer` please? 😄

Reply:
oh lol yeah, me neither, missed this

Reply:
@martin-nhs Fixed